launcher_common/VideoDeviceInfoProviderD3D11.cpp

```diff
@@ -15,7 +15,7 @@ class VideoDeviceInfoProviderD3D11 : public VideoDeviceInfoProvider
     ~VideoDeviceInfoProviderD3D11() override = default;
     std::vector<std::string> get_adapters() override;
     std::set<Resolution> get_resolutions(unsigned adapter, unsigned format) override;
-    std::set<unsigned> get_multisample_types(unsigned adapter, unsigned format, bool windowed) override;
+    std::set<uint32_t> get_multisample_types(uint32_t adapter, uint32_t format, bool windowed) override;
```

Comment:
all other functions use `unsigned adapter` so now code is inconsistent and messy. I suggest to cancel those type changes on method parameters